packages/rancher-monitoring/generated-changes/patch/templates/_helpers.tpl.patch

```diff
@@ -1,13 +1,29 @@
 --- charts-original/templates/_helpers.tpl
 +++ charts/templates/_helpers.tpl
-@@ -1,3 +1,94 @@
+@@ -1,3 +1,110 @@
 +# Rancher
 +{{- define "system_default_registry" -}}
 +{{- if .Values.global.cattle.systemDefaultRegistry -}}
 +{{- printf "%s/" .Values.global.cattle.systemDefaultRegistry -}}
 +{{- end -}}
 +{{- end -}}
 +
++{{/*
++https://github.com/helm/helm/issues/4535#issuecomment-477778391
++Usage: {{ include "call-nested" (list . "SUBCHART_NAME" "TEMPLATE") }}
++e.g. {{ include "call-nested" (list . "grafana" "grafana.fullname") }}
++*/}}
```

Comment:
What is this section? It looks like a templated comment?

Reply:
yes, this is a Helm template style multi-line comment, as described in https://helm.sh/docs/chart_best_practices/templates/#comments-yaml-comments-vs-template-comments